openml/datasets/dataset.py

```diff
@@ -368,9 +368,18 @@ def decode_arff(fh):
     def _convert_array_format(data, array_format, attribute_names):
         """Convert a dataset to a given array format.
 
-        By default, the data are stored as a sparse matrix or a pandas
-        dataframe. One might be interested to get a pandas SparseDataFrame or a
-        NumPy array instead, respectively.
+        Parameters
+        ----------
+        array_format : str
+            Tag to attach to the dataset to get a pandas SparseDataFrame or a
```

Comment:
I am not sure what you mean with "tag to attach to the dataset". From my understanding it would rather read something like "Datatype to convert the data to".
Please also add the list of legal values ('array' and 'dataframe').

Reply:
I believe this was the existing docstring and I was sceptical in removing it altogether so kind of cut-pasted what was there. 
I'll make the changes.